resources/lang/en/admin/components/general.php

```diff
@@ -12,4 +12,5 @@
     'remaining' 			             => 'Remaining',
     'total' 			                 => 'Total',
     'update'                            => 'Update Component',
+    'checkin_limit'            => 'Amount checked in must be equal to or less than this amount'
```

Comment:
This would be a great place to use a variable. Our implementation in the blade is - sub-optimal (and that's my fault), because different languages might be represent values in different parts of a sentence.

Something like:

`'checkin_limit'            => 'Amount checked in must be equal to or less than :assigned_qty`

and then in the blade change (below) something like:

`{{ trans(admin/components/general.checkin_limit, ['assigned_qty' => '$component_assets->assigned_qty']) }}`

It's a minor detail, but comes across a little "friendlier".